clusters/psi/create_kubeconfig.sh

```diff
@@ -22,6 +22,8 @@ API_SERVER_URL=$5
 readonly API_SERVER_URL
 SECRET=$6
 readonly SECRET
+SKIP_TLS_VERIFY=${SKIP_TLS_VERIFY:-false}
```

Comment:
What is the use case here?
In my opinion, we should not join a cluster into CI production before it's URL is CA trusted.

Reply:
By default, its `false`, thus you dont have to pass it for OSD.
This keeps the compatibility with OCP, so we have the option to easily onboard OCP.

> However, we do not plan to add any OCP cluster in any near future.
I am not sure if thats the case. We onboarded two already, more could come...

Reply:
Default value is not needed if we do not plan to use it.